packages/showcase/src/app/components/tab/metadata.ts

```diff
@@ -0,0 +1,514 @@
+            properties: [{ name: "slot", value: "item-4" }]
+          }
+        ]
+      }
+    },
+
+    elevation1: {
+      linkId: "elevation-1",
+      title: "1.10. Elevation 1",
+      states: [
+        {
+          name: "tabModel",
+          type: "TabModel",
+          value: simpleModel
+        }
+      ],
+      imports: [
+        chameleonImportType("TabModel"),
+        'import { getIconPath } from "@genexus/mercury/assets-manager.js";'
+      ],
+      template: {
+        tag: CSS.TAB.TAG,
+        class: CSS.TAB.SELECTOR,
+        properties: [
+          { name: "model", value: "tabModel", state: true },
+          { name: "selectedId", value: "item-1" },
+          { name: "tabListPosition", value: "block-start" }
+        ],
+        children: COMMON_CHILDREN
+      }
+    },
+
+    elevation2: {
+      linkId: "elevation-2",
+      title: "1.11. Elevation 2",
+      states: [
+        {
+          name: "tabModel",
+          type: "TabModel",
+          value: simpleModel
+        }
+      ],
+      imports: [
+        chameleonImportType("TabModel"),
+        'import { getIconPath } from "@genexus/mercury/assets-manager.js";'
+      ],
+      template: {
+        tag: CSS.TAB.TAG,
+        class: `${CSS.TAB.SELECTOR} ${CSS.ELEVATION[2].SELECTOR}`,
+        properties: [
+          { name: "model", value: "tabModel", state: true },
+          { name: "selectedId", value: "item-1" },
+          { name: "tabListPosition", value: "block-start" }
+        ],
+        children: COMMON_CHILDREN
+      }
+    },
+
+    elevation3: {
+      linkId: "elevation-3",
+      title: "1.12. Elevation 3",
+      states: [
+        {
+          name: "tabModel",
+          type: "TabModel",
+          value: simpleModel
+        }
+      ],
+      imports: [chameleonImportType("TabModel")],
+      template: {
+        tag: CSS.TAB.TAG,
+        class: `${CSS.TAB.SELECTOR} ${CSS.ELEVATION[3].SELECTOR}`,
+        properties: [
+          { name: "model", value: "tabModel", state: true },
+          { name: "selectedId", value: "item-1" },
+          { name: "tabListPosition", value: "block-start" }
+        ],
+        children: [
+          {
+            tag: "div",
+            class: CSS.SPACING_BODY.ALL,
+            children: "Item 1 slotted content",
+            properties: [{ name: "slot", value: "item-1" }]
+          },
+          {
+            tag: "div",
+            class: CSS.SPACING_BODY.ALL,
+            children: "Item 2 slotted content",
+            properties: [{ name: "slot", value: "item-2" }]
+          },
+          {
+            tag: "div",
+            class: CSS.SPACING_BODY.ALL,
+            children: "Item 3 slotted content",
+            properties: [{ name: "slot", value: "item-3" }]
+          },
+          ...COMMON_CHILDREN
+        ]
+      }
+    },
+
+    slotTabListStart: {
+      linkId: "slot-tab-list-start",
+      title: "1.13. Slot: Tab List Start",
```

Comment:
I wouldn't use the word "Slot" as not all users will understand its meaning. Perhaps something more practical like `"Additional actions in Tab List: Start"`

Reply:
@ncamera I updated the title to "Customized Tablist: Start" and "Customized Tablist: End" as it is shorter. What do you think?